components/browser/menu2/src/test/java/mozilla/components/browser/menu2/adapter/icons/DrawableMenuIconViewHoldersTest.kt

```diff
@@ -85,7 +84,7 @@ class DrawableMenuIconViewHoldersTest {
     }
 
     @Test
-    fun `async view holder sets icon on view`() = testDispatcher.runBlockingTest {
+    fun `async view holder sets icon on view`() = runTestOnMain {
```

Comment:
These two `runTestOnMain` here seem no needed at all, right?

Reply:
In this tests `MainCoroutineRule` was already used.
So any `runBlocking` / `runBlockingTest` usages were directly moved to use `runTestOnMain`.